ext/openssl/openssl.c

```diff
@@ -1176,6 +1195,18 @@ PHP_MINIT_FUNCTION(openssl)
 	REGISTER_LONG_CONSTANT("PKCS7_BINARY", PKCS7_BINARY, CONST_CS|CONST_PERSISTENT);
 	REGISTER_LONG_CONSTANT("PKCS7_NOSIGS", PKCS7_NOSIGS, CONST_CS|CONST_PERSISTENT);
 
+
+	REGISTER_LONG_CONSTANT("CMS_DETACHED", CMS_DETACHED, CONST_CS|CONST_PERSISTENT);
```

Comment:
I know that some previous constant (like PKCS7 ones) don't do that but could we prefix the new ones with `OPENSSL` (in this case `OPENSSL_CMS_DETACHED`)

Reply:
Ok.